xDrip/Scenes/Entries/EntriesList/EntriesListPresenter.swift

```diff
@@ -19,10 +19,38 @@ protocol EntriesListPresentationLogic {
 final class EntriesListPresenter: EntriesListPresentationLogic {
     weak var viewController: EntriesListDisplayLogic?
     
+    private var dateFormatter: DateFormatter {
+        let formatter = DateFormatter()
+        formatter.locale = Locale(identifier: "en_US_POSIX")
+        formatter.dateFormat = "dd.MM.YYYY HH:mm"
+        
+        return formatter
+    }
+    
     // MARK: Do something
     
     func presentLoad(response: EntriesList.Load.Response) {
-        let viewModel = EntriesList.Load.ViewModel()
+        let responseData = response.entries
+        var cellData: [EntriesList.CellData] = []
+        
+        responseData.forEach { (entry) in
+            var value = ""
+            
+            if response.type == .bolus {
```

Comment:
The same approach should be followed, as in the case of interactor. Create an abstract formatting protocol, e.g.
```swift
protocol EntriesListFormattingWorker {
  func formatEntries(_ entries: [AbstractEntry]) -> [EntriesList.CellViewModel]
}
```

Create concrete implementations, inject them etc